sql-plugin/src/main/spark311/scala/org/apache/spark/sql/hive/rapids/shims/CommandUtilsShimBase.scala

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
We don't need this trait but if you still want to keep it why not just keep it outside in the common source? Is it not applicable to a certain version of Spark?

Reply:
Ok, that's doable.  This is the fallout from my reaction to the feedback in #10857. :)) I'll remove the base.